package.json

```diff
@@ -98,8 +99,8 @@
     "@types/react": "^18.2.21",
     "@types/react-dom": "^18.2.7",
     "@types/react-virtualized": "^9.21.22",
-    "@typescript-eslint/eslint-plugin": "^5.59.11",
-    "@typescript-eslint/parser": "^5.59.11",
+    "@typescript-eslint/eslint-plugin": "^6.7.2",
+    "@typescript-eslint/parser": "^6.7.2",
```

Comment:
We only want these changes to address updating these two:
```
"@typescript-eslint/eslint-plugin": "^6.7.2",
"@typescript-eslint/parser": "^6.7.2",
```

Can you please remove the other updates from this file so we can isolate the changes here and better test what has been updated. Thank you!